### PR TITLE
Updated API documentation and examples

### PR DESCRIPTION
**Pull request by mattray**

### CODE REVIEW

File: docs/integrations/api-examples.md [hunk 1]
```diff
@@ -84,6 +84,248 @@ curl http://localhost:9003/allocation/compute \
...
+
+### Default OpenCost UI Cloud Cost request
+
+Amortized Net Cost per Provider for the last 7 days
+
+```sh
+curl -G http://localhost:9003/cloudCost \
+  -d window=7d \
+  -d aggregate=provider \
+  -d costMetric=AmortizedNetCost
```
Comment (Sean-Holcomb) [inline docs/integrations/api-examples.md:99]: costMetric is not a valid input for `/cloudCost`

File: docs/integrations/api.md [hunk 2]
```diff
@@ -104,84 +104,74 @@ The Cloud Costs API retrieves cloud cost data from cloud providers by reading co
...
-<td/>
-<td>
-<a href="https://docs.kubecost.com/apis/apis-overview/filters-api">V2 filter parameter</a>.
-<br/><br/>
-Examples:<br/>
-<ul>
-<li></li>
-</ul>
-</td>
-</tr>
```
Comment (Sean-Holcomb) [inline docs/integrations/api.md:184]: These should not be removed

File: docs/integrations/api.md [hunk 3]
```diff
@@ -104,84 +104,74 @@ The Cloud Costs API retrieves cloud cost data from cloud providers by reading co
...
+    <br/><br/>
+    Examples:<br/>
+      <ul>
+        <li><code>costMetric=AmortizedNetCost</code> - Returns the Amortized Net Costs.</li>
+        <li><code>costMetric=InvoicedCost</code> - Returns the Invoiced Costs.</li>
+        <li><code>costMetric=ListCost</code> - Returns the List Costs.</li>
+        <li><code>costMetric=NetCost</code> - Returns the Net Costs.</li>
+      </ul>
+    </td>
+  </tr>
```
Comment (Sean-Holcomb) [inline docs/integrations/api.md:174]: This is not valid for this endpoint

**Review by Sean-Holcomb — COMMENTED** [review]
Do we want to include the View API in this PR?

File: docs/integrations/api.md (hunk 2 above)
Comment (mattray) [reply docs/integrations/api.md:184]: restored

File: docs/integrations/api.md (hunk 3 above)
Comment (mattray) [reply docs/integrations/api.md]: removed

**Comment by mattray** [comment]
@Sean-Holcomb made the updates based on your feedback. There have been a couple other API changes made that need to be documented, so this is a first of many and I'll add the other /cloudCost endpoints in a subsequent PR

**Review by Sean-Holcomb — APPROVED** [review]

**Pull request merged**